pkg/codegen/hcl2/binder_test.go

```diff
@@ -1,51 +0,0 @@
-package hcl2
```

Comment:
This file is deleted because the functionality tested here is already tested via the programgen tests and it was causing an import cycle error.

Reply:
Can we restore these tests now that the cycle has been broken?